src/App.jsx

```diff
@@ -1,35 +1,27 @@
 import './App.css';
-import { useEffect, useState } from 'react';
+import { useState } from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
-const useSemiPersistentState = () => {
 
-  const [todoList, setTodoList] = useState( JSON.parse(localStorage.getItem('list')) ||  []);
-  
-  useEffect( () => {
-    
-    localStorage.setItem('list', JSON.stringify(todoList)) 
-  }, [todoList]) // пустой зависимый массив
-  return [todoList, setTodoList];
-} 
+const App = () => {
+  const [todoList, setTodoList] = useState([]);
 
-function App() {
-    const [todoList, setTodoList] = useSemiPersistentState(); // без аргумента
-
-    const addTodo = (newTodo) => {
-        setTodoList((prevTodoList) => [...prevTodoList, newTodo]);
+    const addTodo = (title) => {
+      setTodoList([...todoList, { id: Date.now(), title }]);
```

Comment:
Please revert to the previous version of the addTodo function because now the todo is not being added to your list.

```
 const addTodo = (newTodo) => {
      setTodoList([...todoList, newTodo]);
    };
```
or 

```
 const addTodo = (newTodo) => {
     setTodoList((prevTodoList) => [...prevTodoList, newTodo]);
    };
```